src/mail.sh

```diff
@@ -99,6 +99,7 @@ function mail_send()
   local kernel_root
   local patch_count=0
   local cmd='git send-email'
+  declare -g patch_cache="${KW_CACHE_DIR}/patches"
```

Comment:
Tbh, I don't see the value of making this variable global and in this function. If you think we can reuse it in multiple areas and we do not change it, declare it at the beginning of the file using upper case and the read-only option. Making this variable global inside this function is confusing.

Reply:
I also think that this is the case of making the variable global in the beginnig of the file, but for some reason, "$KW_CACHE_DIR" is an empty var in the begining of the code and i didn't understand where it's value is updated inside this code